custom/stdweb/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "getrandom-stdweb"
+version = "0.1.0"
+edition = "2018"
+authors = ["The Rand Project Developers"]
+license = "MIT OR Apache-2.0"
+description = "Custom shim for using getrandom with stdweb"
+documentation = "https://docs.rs/getrandom-stdweb"
+repository = "https://github.com/rust-random/getrandom/tree/master/custom/stdweb"
+categories = ["wasm"]
+
+[dependencies]
+getrandom = { path = "../..", features = ["custom"] }
+stdweb = "0.4.18"
+log = "0.4"
+
+# Unstable features used to build tests correctly
+[features]
+default = ["test-stdweb"]
```

Comment:
It's a bit strange to enable an unstable feature by default. Can't we do it in CI job somehow?

Reply:
We could, but then a users running `cargo test` on these repos would have things break.

These features only affect tests, so it's not that bad. Ideally, we would have a way to enable these features in tests. I'll look into https://github.com/rust-lang/cargo/issues/2911 and `required-features`.

Reply:
I changed the comments to clarify that these are non "unstable" but that they are only for tests. I'm actually fine with the features being stable so that "cargo test" just works.

We could use `required-features` here to skip the tests if you haven't enabled the features, but that would just make the tests appear to pass when the features aren't set. To actually solve this we would need https://github.com/rust-lang/cargo/issues/2911. Having the test-only features on by default seems like the easiest for users and doesn't hurt anything.